base/docs/helpdb/Base.jl

```diff
@@ -549,26 +549,24 @@ to synchronous `File`'s and `IOStream`'s not to any of the asynchronous streams.
 """
 fd
 
+
 """
-    ones(type, dims)
+    ones([A::AbstractArray,] [T=eltype(A)::Type,] [dims=size(A)::Tuple])
 
-Create an array of all ones of specified type. The type defaults to `Float64` if not specified.
+Create an array of all ones with the same layout as `A`, element type `T` and size `dims`.
+The `A` argument can be skipped, which behaves like `Array{Float64,0}()` was passed.
+For convenience `dims` may also be passed in variadic form.
```

Comment:
I think "variadic form" is fine, but it should be spelled out with an example as well.